www/guanlecoja-ui/src/module/topbar/topbar.less

```diff
@@ -63,17 +63,10 @@
         padding-left: 0px;
     }
 }
-.avatar {
```

Comment:
Not sure why this is declared twice. It gets output twice too.

I'm not sure which would be the better one to remove so I kept them in sync.

Reply:
I think this is from the time when guanlecoja-ui was a separate project. After it got merged to Buildbot, only minimal cleanup has been done.